modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/sql/IgniteAbstractSqlAlterZone.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.sql.engine.sql;
+
+import org.apache.calcite.sql.SqlAlter;
+import org.apache.calcite.sql.SqlKind;
+import org.apache.calcite.sql.SqlOperator;
+import org.apache.calcite.sql.SqlSpecialOperator;
+import org.apache.calcite.sql.parser.SqlParserPos;
+
+/**
+ * A base class for a parse tree for {@code ALTER ZONE } statement.
+ */
+public abstract class IgniteAbstractSqlAlterZone extends SqlAlter {
+
+    /** Alter operator. */
+    private static final SqlOperator OPERATOR =
+            new SqlSpecialOperator("ALTER ZONE", SqlKind.OTHER_DDL);
+
+    /** Scope. **/
+    private static final String SCOPE = "ZONE";
+
+    /** Constructor. */
+    public IgniteAbstractSqlAlterZone(SqlParserPos pos) {
```

Comment:
seems **public** access is redundant here

Reply:
Fixed